packages/vue/src/components/basic/SelectedFilters.jsx

```diff
@@ -67,7 +71,43 @@ const SelectedFilters = {
 			this.$emit('clear', component, value);
 		},
 		clearValues() {
-			this.clearValuesAction();
+			const { resetToDefault, componentProps, selectedValues } = this;
+			if (resetToDefault) {
+				let valueToSet;
+				Object.keys(selectedValues).map(component => {
```

Comment:
@mohdashraf010897 Where are you using `mapped` value? Use `forEach` instead

Reply:
👍